{{cookiecutter.repo_name}}/docs/conf.py

```diff
@@ -76,7 +76,7 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'alabaster'
+html_theme = 'sphinx_rtd_theme'
```

Comment:
This isn't quite sufficient. You will need [these](https://github.com/dgasmith/EEX_scratch/blob/master/docs/source/conf.py#L130-L151) lines as well I believe.

Reply:
It is for me? I have to have the `sphinx_rtd_theme` package installed. But whether I do it my way, or your way, the output is the same, at least for this cookiecutter. 

We do the same thing for [`yank`](https://github.com/choderalab/yank/blob/master/docs/conf.py#L124-L136) where all I have done is [extend the stock `sphinx_rtd_theme`](https://github.com/choderalab/yank/blob/master/docs/versions_div_theme/theme.conf#L1-L3), hence the extra options.

It might have something to do with `which` version of sphinx you had at the time you needed that? I have found in the times since I started in on YANK's docs the Sphinx versions were subtly, but obnoxiously different.

Reply:
Ah, if you have it solved with pinned sphinx versions then it works for me. I rarely setup RTD and rarely go back the setup. So I usually just hunt around until it works.

Reply:
> So I usually just hunt around until it works.

As it goes with Sphinx. I was not going to worry about pinning Sphinx here as any new project derived from this should not have this problem. The `meta.yaml` file does not even include `sphinx` since its a doc-specific thing and not needed for the base package. 

I left all the instructions in the `docs/README.md` file for if/when people want to start working on their docs. I could change that though if others think we should more sternly enforce doc building.